tests/test_consistency.py

```diff
@@ -3,74 +3,87 @@
 import accern_data
 import pandas as pd
 import pandas.testing as pd_test
+import pytest
 from accern_data.accern_data import DT_FORMAT
 from accern_data.util import load_json
 
 
-def test_csv_full_csv_date_consistency() -> None:
+@pytest.mark.parametrize("sheet_mode", ["csv", "df"])
+def test_csv_full_csv_date_consistency(sheet_mode: str) -> None:
     start_date = "2022-01-03"
     end_date = "2022-03-04"
     output_pattern = "test_csv_full_csv_date_consistency"
+    output_path = "./tests/outputs/"
     client = accern_data.create_data_client(
         "http://api.example.com/", "SomeRandomToken")
-    client.set_mode("csv_date")
+    client.set_mode(sheet_mode, split_dates=True)
     client.download_range(
         start_date=start_date,
-        output_path="./tests/outputs/",
+        output_path=output_path,
         output_pattern=output_pattern,
         end_date=end_date,
+        mode=sheet_mode,
```

Comment:
it's not a test of the functionality if you set it both here and above.
set it only in one place unless you want to test that overwrite functionality and in that case you want to use a different value to see if it works correctly